documentation and version number bump (#64)
amzn/ion-vscode-plugin: README.md

# VSCode extension for Ion 

> VSCode extension (plugin) that supports the Ion data format. You can download this extension from VSCode marketplace. 

Reviewer: ```suggestion
[VSCode extension (plugin)](https://marketplace.visualstudio.com/items?itemName=TheIonTeam.ion-extension) that supports the Ion data format.
```

> GitHub repository: https://github.com/amzn/ion-vscode-plugin 

Reviewer: This is a link to the page they are already on :). Remove please.

## Functionality 

- **Syntax highlighting** 

>     Colorize the text document. 

Reviewer: ```suggestion
    Colorize the Ion values in your document. 
```

-  **Error reporting** 

>     Give a Squiggly line for the error part. For now, it only finds the first error in the editor. 

Reviewer: ```suggestion
    Reports errors in the editor and the "Problems" tab. Currently supports whole document validation. 
```

- **Formatting** 

>     Press `Shift + Option + F`, then type `Format Document`. 

Reviewer: ```suggestion
    Formats the whole document. Currently support whole error-free document formatting. 
```

# VSCode extension for Ion 

[VSCode extension (plugin)](https://marketplace.visualstudio.com/items?itemName=TheIonTeam.ion-extension) that supports the Ion data format.

## Functionality 

- **Syntax highlighting** 
    Colorize the Ion values in your document. 
-  **Error reporting** 
    Reports errors in the editor and the "Problems" tab. Currently supports whole document validation. 
- **Formatting** 

>     Formats the whole document. Currently support whole error-free document formatting. 

Reviewer: ```suggestion
    Formats the whole document. Currently supports whole error-free document formatting. 
```

> Usage: Press `Shift + Option + F`, then select `Format Document`. 

Reviewer: This is VSCode standard, it is also available through the VScode menu I think we can remove this line

## Functionality 

* Syntax highlighting. 
* Error reporting. 

> * Formatting. 

Reviewer: We should add a sentence to each item that this happens on the whole file and not while typing for now.

## Development 

> 

Reviewer: We should also add the instructions on how to check out the repo. You can reuse the text and from one of our other libs here.
